packages/opencensus-instrumentation-mongodb/package.json

```diff
@@ -49,6 +49,7 @@
     "access": "public"
   },
   "devDependencies": {
+    "@types/bson": "1.0.11",
```

Comment:
This was needed because `@types/mongodb` doesn't have a specifier for its dependnecy on this package. This makes it pull the same version that the actual `mongodb` package does.